src/commands/deploy.rs

```diff
@@ -90,14 +95,39 @@ pub struct DeployCommand {
         env = "HIPPO_PASSWORD"
     )]
     pub hippo_password: String,
+
+    /// Disable attaching buildinfo
+    #[clap(long = "no-buildinfo", conflicts_with = "buildinfo")]
+    pub no_buildinfo: bool,
+
+    /// Build metadata to append to the bindle version
+    #[clap(
+        name = BUILDINFO_OPT,
+        long = "buildinfo",
+        parse(try_from_str = parse_buildinfo),
+    )]
+    pub buildinfo: Option<BuildMetadata>,
+
+    /// Re-deploy bindle if it already exists in bindle server
```

Comment:
Is this doc string accurate?  It looks like all the `redeploy` option does is suppress the error report if the bindle ID already exists.  If I make a change to my application and do a `deploy --redeploy`, it won't actually upload the changes will it?  (Forgive me if I'm wrong, I'm having to go on just reading the code so may be missing something!)

(sorry wrote this yesterday but forgot to push the button)

Reply:
My read was that it allows `create_and_push_bindle` to succeed on duplicate bindle rather than failing and short-circuiting the deploy.